modules/ROOT/pages/to-deploy-using-rest-connect.adoc

```diff
@@ -182,20 +182,24 @@ baseUri: https://jsonplaceholder.typicode.com
 version: 0.1
 mediaType: application/json
 
+uses:
+  rest-connect: exchange_modules/org.mule.connectivity/rest-connect-library/1.0.0/rest-connect-library.raml
+
   ...
   /{postId}:
     uriParameters:
       postId: integer
 
     get:
+      (rest-connect.operationName): Retrieve a post by id
       displayName: Get a post by ID.
       responses:
         200:
           body:
             type: Post
 ----
 
-Example with REST Connect library:
+You can also change the text in `displayName`, for example:
```

Comment:
IMHO this method is the easiest and standard, maybe it should be offered first as it was.